src/control/cmd/dmg/storage.go

```diff
@@ -264,7 +264,7 @@ type nvmeSetFaultyCmd struct {
 // Execute is run when nvmeSetFaultyCmd activates
 // Set the SMD device state of the given device to "FAULTY"
 func (cmd *nvmeSetFaultyCmd) Execute(_ []string) error {
-	cmd.Info("WARNING: This command will permanently mark the device as unusable!")
+	cmd.Notice("WARNING: This command will permanently mark the device as unusable!")
```

Comment:
Should we modify the `Notice()` formatter to automatically prepend `NOTICE:` to messages? It seems a little inconsistent to have some messages saying WARNING and not others.

Reply:
Given that this ticket will fix a currently-broken test on master, I think we should land it now and maybe address this in a follow-on. I do think it would be good for the messages to be consistent and not up to the developer.

Reply:
I tend to agree. I opted to leave the messages as-is for now, but I can add the prefix.